setup.py

```diff
@@ -12,7 +12,7 @@
     include_dirs  = ['http-parser', '/usr/include/libev'],
     define_macros = [('WANT_SENDFILE', '1'),
                      ('WANT_SIGINT_HANDLING', '1')],
-    extra_compile_args = ['-std=c99', '-fno-strict-aliasing', '-fcommon',
+    extra_compile_args = ['-std=c11', '-fno-strict-aliasing', '-fcommon',
```

Comment:
Makefile error:

hua@grass:~/git/bjoern $ make
/bin/sh: 1: --exec-prefix: not found
/bin/sh: 1: --includes: not found
/bin/sh: 1: --cflags: not found
/bin/sh: 1: --libs: not found
/bin/sh: 1: /bin/sh: 1: --extension-suffix: not found
--ldflags: not found/bin/sh: 1:
--configdir: not found
/bin/sh: 1: --help: not found
Makefile:52: recipe for target 'build/filewrapper.o' failed
make: *** [build/filewrapper.o] Error 127
hua@grass:~/git/bjoern $